ext_emconf.php

```diff
@@ -11,7 +11,7 @@
     'constraints' => [
         'depends' => [
             'php' => '7.4.0-8.3.99',
-            'typo3' => '11.5.0-11.5.99',
+            'typo3' => '11.5.0-12.9.99',
```

Comment:
going up to x.9.99 is something we did back when LTS did not exist, because nobody knew how far it would go. We know that 12 LTS is 12.4.x, thus it should read 12.4.99